### DISCUSSION TITLE
Welcome to VelocityX Discussions!

### DISCUSSION THREAD

**Discussion by iampawan** [iampawan/VelocityX#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JoDeveloper** [comment]
@iampawan
Thanks a lot for `velocity_x ` it's really amazing 🌹

I'm wondering why did you mixed it with `state-mangement ` would it be better if it just a UI framework

❤️

**Comment by iampawan** [comment]
The only reason to mix it is that VelocityX was made for faster development
and since we already have many utilities so state management in Vx also
works just like another utility, however it’s totally optional to use.

**Comment by zaid-bashir** [comment]
Thanks Pawan Sir to have such a great flutter framework.I have started using your framework VelocityX and it was such a great fun.It decreased my development time to such a great extent  in creating clean and amazing flutter cross-platform apps.
THANKS...
I am sure sir that you keep on making new releases of VelocityX.

**Comment by rajsinh2181** [comment]
I have completely watched your video for Catalog App. It's quite amazing and helpful. Keep up the good work. 
If you could possibly integrate a database to the app, it will be quite helpful for the beginners.
thank you.

**Comment by gvjkumar2004** [comment]
@iampawan ![image](https://user-images.githubusercontent.com/108649499/212150427-571460aa-6478-4fff-b775-7322c55e4311.png)
sir i am getting this error while using consumer, can u help me get out of this....
